firefox-ios/Client/Frontend/Browser/BrowserViewController/Views/BrowserViewController.swift

```diff
@@ -2792,14 +2793,17 @@ extension BrowserViewController: LegacyTabDelegate {
                             method: .view,
                             object: .loginsAutofillPromptShown
                         )
+                    } else {
```

Comment:
Nit: I learned that it's better to start with the positive case. So instead of checking for `if !loginsForCurrentTab.isEmpty`, check `if loginsForCurrentTab.isEmpty` first. Basically the if would be switched around.